src/apps/myaccount/submodules/transactions/transactions.js

```diff
@@ -150,9 +150,10 @@ define(function(require) {
 						.value();
 
 					args.callback({
-						amount: _.reduce(formatted, function(acc, transaction) {
-							return acc + parseFloat(transaction.amount);
-						}, 0),
+						amount: _
+							.chain(formatted)
+							.filter('isApproved')
+							.sumBy('amount'),
```

Comment:
Although it works in this particular case (thanks to the [type casting power](https://github.com/2600hz/monster-ui/blob/master/src/js/lib/monster.util.js#L954) of `monster.util#formatPrice()`), the result of a chain sequence should always be unwrapped using `_#value` to retrieve the actual value, or else you will end up with the lodash wrapper instance.
```suggestion
							.sumBy('amount')
							.value(),
```

Reply:
oups, thanks